Source/Transmitters/LMCDipoleAntenna.cc

```diff
@@ -11,7 +11,8 @@
 namespace locust
 {
 
-    DipoleAntenna::DipoleAntenna()
+    DipoleAntenna::DipoleAntenna():
+    fMomentVector( 0., 0., 1.0 )
```

Comment:
I think it makes sense for this to be configurable. For example, this moment is (1,0,0) in Maurio's dipole case.